wolfssl/ssl.h

```diff
@@ -674,6 +674,12 @@ enum AlertLevel {
     alert_fatal   = 2
 };
 
+#if defined(WOLFSSL_ERROR_CODE_OPENSSL)
```

Comment:
Please add a comment here about what WOLFSSL_ERROR_CODE_OPENSSL does and why we need it.  This will help future engineers understand what it is doing.

Reply:
Added comment for RETURN_CODE macro.

Reply:
Thanks